<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Quitemorph</class>
 <widget class="QWidget" name="Quitemorph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>259</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Quitemorph</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>854</width>
     <height>263</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="g_button">
      <property name="text">
       <string>G</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="g_path"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="h_button">
      <property name="text">
       <string>H</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="h_path"/>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Начальная раскраска:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>сброшена</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>из файлов</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>После стабилизации раскраски:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="poststable_mode">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>ничего (сравнить полученное)</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="placeholderText">
         <string>Выбор режима</string>
        </property>
        <item>
         <property name="text">
          <string>ничего (сравнить полученное)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>подобрать перестановку перебором</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>дестабилизировать и продолжать</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Дополнительно:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="graphics">
        <property name="text">
         <string>графика</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="timings">
        <property name="text">
         <string>время работы</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>пошагово</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="output_button">
      <property name="text">
       <string>Вывод</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="output_path"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="start_button">
      <property name="text">
       <string>Запуск</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
